lib/index.ts

```diff
@@ -69,7 +82,16 @@ function searchBabylonFiles(root: string, currentPath: string, options: SearchOp
                 const id = geometry.id;
 
                 if (!doNotDelayLoadingForGeometries.some(g => g === id)) {
-                    extract(geometry, currentPath, filename, false);
+
+                    const geometryString = createDelayLoadingString(geometry, false);
+                    if (!minMeshSize || geometryString.length > minMeshSize) {
+                        extract(geometry, currentPath, filename, false, geometryString);
+                    } else {
+                        // tslint:disable-next-line:no-console
+                        console.log(
+                            `Skipping ${geometry.id} as size ${geometryString.length} is smaller then ${minMeshSize}`,
```

Comment:
Typo, then instead of than. It would be nice to also mention where the number ${minMeshSize} comes from, e.g. Skipping ${mesh.name} as size ${meshString.length} is smaller than ${minMeshSize} from minMeshSize option